pkg/minikube/bootstrapper/bsutil/files.go

```diff
@@ -59,3 +62,23 @@ func CopyFiles(runner command.Runner, files []assets.CopyableFile) error {
 	}
 	return nil
 }
+
+// ReverseDirList returns a list of subdirectories under the given path, sorted in reverse order, and any error.
+// If n > 0, ReverseDirList returns at most n subdirectories.
+// If n <= 0, ReverseDirList returns all the subdirectories from the directory.
+func ReverseDirList(path string, n int) (list []string, err error) {
```

Comment:
seems like this function is only used in the test ? then let it put in the test file
also if func not used outside the package, it must be private 
also comment why we need this function ? what it is used for ?

Reply:
@medyagh all good points, thank you: i've merged ReverseDirList() into recentReleases() and then reused it where needed